packages/zosmf/__tests__/__system__/api/methods/ListDefinedSystems.system.test.ts

```diff
@@ -0,0 +1,154 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ListDefinedSystems, ZosmfMessages } from "../../../../../zosmf";
+import { Session, Imperative } from "@brightside/imperative";
+import { inspect } from "util";
+import { ITestEnvironment } from "../../../../../../__tests__/__src__/environment/doc/response/ITestEnvironment";
+import { TestProperties } from "../../../../../../__tests__/__src__/properties/TestProperties";
+import { ITestSystemSchema } from "../../../../../../__tests__/__src__/properties/ITestSystemSchema";
+import { TestEnvironment } from "../../../../../../__tests__/__src__/environment/TestEnvironment";
+
+let REAL_SESSION: Session;
+let testEnvironment: ITestEnvironment;
+let systemProps: TestProperties;
+let defaultSystem: ITestSystemSchema;
+
+describe("List Defined Systems Api", () => {
+    beforeAll(async () => {
+        testEnvironment = await TestEnvironment.setUp({
+            tempProfileTypes: ["zosmf"],
```

Comment:
You don't need temp profile types for an API test by the way. It's designed for CLI tests that use the runCliScript functionality